### PR TITLE
Improve ACLs

### PR DESCRIPTION
**Pull request by gber**
* Fix accidental search-and-replace error
* Rename token* -> addToken*
* Split mfaRequired into mfaRequiredByRule and mfaRequiredByUser

### CODE REVIEW

File: personal/privacyidea/class_mfaAccount.inc [hunk 1]
```diff
@@ -512,8 +512,8 @@ class mfaAccount extends plugin
 
         $this->smarty->assign("allTokenTypes", array_keys($this->mfaTokens));
 
-        $mfaRequiredByUser = $this->isCurrentUserUsertab() ? ($this->read_only ? "r" : "rwcdm") : $this->getacl("mfaRequired");
-        $this->smarty->assign("mfaRequiredACLorCurrentUser", $mfaRequiredByUser);
+        $mfaRequiredByUser = $this->isCurrentUserUsertab() ? ($this->read_only ? "r" : "rwcdm") : $this->getacl("mfaRequiredByUser");
+        $this->smarty->assign("mfaRequiredByUserACLorCurrentUser", $mfaRequiredByUser);
```
Comment (sunweaver) [inline personal/privacyidea/class_mfaAccount.inc:516]: The CurrentUser bit should be coverable by an ACL, as well. In GOsa² you can mark UI elements / attributes as editable (or whatever perm) by the owner of the object. So we very don't need the currentUser check here.

**Review by sunweaver — CHANGES_REQUESTED** [review]
This probably can be simplified even more by getting rid of the isCurrentUser check.

File: personal/privacyidea/class_mfaAccount.inc (hunk 1 above)
Comment (gber) [reply personal/privacyidea/class_mfaAccount.inc]: The special casing for the current user has been removed.

**Comment by dzatoah** [comment]
Is this still a draft, or may I merge?

**Comment by gber** [comment]
> Is this still a draft, or may I merge?

Feel free to review and merge.

File: personal/privacyidea/class_mfaAccount.inc [hunk 2]
```diff
@@ -705,19 +690,20 @@ class mfaAccount extends plugin
...
+                "addTokenWebAuthn"     => _("Add Security key"),
+                "addTokenTotp"         => _("Add Authentication app (TOTP)"),
+                "tokenDescription"     => _("Description"),
+                "tokenLastUsed"        => _("Last use"),
+                "tokenFailCount"       => _("Failed login counter"),
+                "tokenStatus"          => _("Status"),
+                "tokenRevocation"      => _("Revocation allowed"),
+                "tokenRemoval"         => _("Removal allowed"),
             ),
             // TODO: Outsource ACLs overrideAllowToken_* to MFAToken classes!!
```
Comment (dzatoah) [inline personal/privacyidea/class_mfaAccount.inc:708]:
```suggestion
            // TODO: Outsource addToken* ACLs into the specific MFAToken class (Create getACLInformation() func?)
```

**Comment by dzatoah** [comment]
Thanks, LGTM!

**Pull request merged**